documentation/guide-configuration.asciidoc

```diff
@@ -196,7 +197,22 @@ To prevent jasypt to throw an exception in dev or test scenarios simply put this
 jasypt.encryptor.password=none
 ```
 
-Is this Security by Obscurity?
+==== Configure Algorithm
+Algorithm can be configured for stronger encryption. Jasypt uses PBEWITHHMACSHA512ANDAES_256 as the Default encryption algorithm.
+Algorithm can be configured through System, properties file, command line argumments, environment variable etc.
+
+[source, java]
+----
+StandardPBEStringEncryptor encryptor = new StandardPBEStringEncryptor();
+String privateData = "secret-password-to-encrypt";
+encryptor.setPassword("master-password");
+encryptor.setAlgorithm("PBEWithMD5AndTripleDES");
+String encryptedText = encryptor.encrypt(privateData);
+----
+
+Other configurable properties can be found https://github.com/ulisesbocchio/jasypt-spring-boot#password-based-encryption-configuration[here]
```

Comment:
Very nice improvement. Still it is unclear what to do? Which algorithm should we use now?